### PR TITLE
Remove hostpath

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 
 REGISTRY_NAME=quay.io/k8scsi
-IMAGE_NAME=hostpathplugin
```
Comment (msau42) [inline Makefile]: nfs and iscsi plugins have also moved to their own repo.  @pohly can you help make sure the image building is setup properly in the new repos, and then we can just remove all of the drivers in this PR?

Comment (pohly) [reply Makefile]: Okay. See https://github.com/kubernetes-csi/csi-driver-nfs/pull/5 and https://github.com/kubernetes-csi/csi-driver-iscsi/pull/6.

Comment (humblec) [reply Makefile]: @msau42 @pohly what about moving one by one or in a small group , instead of everything in a PR ?

**Comment by humblec** [comment]
@msau42 Sure, I have removed makefile from the PR. PTAL

**Comment by msau42** [comment]
Can you also modify the top level README to say that this repo is deprecated. And point to the new replacement repos?

I also need to turn off travis for this repo.

**Comment by msau42** [comment]
/retest
I've disabled travis on this repo

**Pull request closed without merging**